src/containers/header/Header.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import './header.css';
+import Image from 'next/image';
+import satellite from '../../assets/satellite.png';
+const Header = () => {
+  return (
+    <div className="scheduler_header section__padding" id="home">
+    <div className="scheduler_header-content">
+      <h1 className="gradient__text">Schedule In Between Space and Earth</h1>
```

Comment:
from my understanding, we dont really care what the public sees, since this is gonna be hosted on their server, so maybe no email signup? just a get started page or something more functional rather than advertising the product